Makefile

```diff
@@ -56,4 +63,9 @@ docker-build: build
 docker-push:
 	docker push ${IMAGE}
 
+deploy: docker-build docker-push
+	sed 's#\(image:\) .*#\1 $(IMAGE)#' /tmp/deis-$(SHORT_NAME) | kubectl apply --validate=true -f -
```

Comment:
At least in kubectl version v1.2.4, I get `Error: unknown command "set" for "kubectl"`